infra/gp-multena/templates/03-deployment.yaml

```diff
@@ -0,0 +1,118 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "gp-multena.name" . }}-proxy
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{- include "gp-multena.labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "gp-multena.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "gp-multena.selectorLabels" . | nindent 8 }}
+    spec:
+      securityContext:
```

Comment:
das sollte in openshift alles default sein. würde ich dementsprechend nicht fix vorgeben. und wenn dann über values konfigurierbarmachen anstatt hardcoded zu defnieren. 
für das manuelle setzten des seccompProfiles benötigst du nämlich privileges die du per default nicht auf dem serviceaccount hast.